packages/theme/components/layout/root-layout/Theme.tsx

```diff
@@ -149,22 +149,43 @@ export function Theme({pageMap, children}: ThemeProps) {
                                     {activeHeaderLink ? activeHeaderLink.title : siteTitle}
                                   </Breadcrumbs.Item>
                                 )}
-                                {activePath.map((item, index) => {
-                                  return (
-                                    <Breadcrumbs.Item
-                                      as={NextLink}
-                                      key={item.name}
-                                      href={item.route}
-                                      selected={index === activePath.length - 1}
-                                      sx={{
-                                        textTransform: 'capitalize',
-                                        color: 'var(--brand-InlineLink-color-rest)',
-                                      }}
-                                    >
-                                      {item.title.replace(/-/g, ' ')}
-                                    </Breadcrumbs.Item>
-                                  )
-                                })}
+                                {activePath
+                                  .filter((item, index, array) => {
+                                    const nextItem = array[index + 1]
+
+                                    // Skip when current item is a folder followed by its index page
+                                    // or when it is an index page with a tabbed navigation
+                                    return !(
+                                      (index < array.length - 1 &&
+                                        nextItem.name === 'index' &&
+                                        item.route === nextItem.route &&
+                                        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+                                        !nextItem.frontMatter?.['tab-label']) ||
```

Comment:
I'm not sure how to avoid disabling the linter here, the fontMatter check is actually necessary.

Reply:
If frontMatter is optional, we need to adjust the types in types.ts to reflect that. But that will likely lead to a bigger job so in that case, let's leave as-is and mop up later.